Sources/Backtrace/Backtrace.swift

```diff
@@ -61,10 +61,25 @@ private let errorCallback: CBacktraceErrorCallback? = {
     }
 }
 
+private func printBacktrace(signal: CInt) {
+    _ = fputs("Received signal \(signal). Backtrace:\n", stderr)
+    backtrace_full(state, /* skip */ 0, fullCallback, errorCallback, nil)
+}
+
 public enum Backtrace {
+    /// Install the backtrace handler on `SIGILL`.
     public static func install() {
-        self.setupHandler(signal: SIGILL) { _ in
-            backtrace_full(state, /* skip */ 0, fullCallback, errorCallback, nil)
+        self.setupHandler(signal: SIGILL) { signal in
```

Comment:
make DRYer by calling the new install func?

Reply:
done!